spec/services/import_transactions_spec.rb

```diff
@@ -399,19 +331,18 @@
 
     it "assigns each transaction to the correct report" do
       expect(report.transactions.map(&:description).sort).to eq([
-        "Christmas Donation",
-        "Fees for Q3",
-        "Rent Payments",
+        "Q4 1999-2000 spend on Example Project",
+        "Q4 1999-2000 spend on Sibling Project",
+        "Q4 1999-2000 spend on Sibling Project",
       ])
     end
 
     it "assigns each transaction to the correct activity" do
-      expect(project.transactions.map(&:description)).to eq([
-        "Rent Payments",
+      expect(project.transactions.map(&:description).uniq).to eq([
```

Comment:
This is testing the association between Transaction and Activity gets set up, so we could avoid having to update the strings in the future:

```ruby
expect(project.transactions).to be_present
expect(sibling_project.transactions).to be_present
```

Reply:
I'm not sure I understand why it would be enough to test that some transactions are present. How are we sure they are, as the spec says "to the correct activity"? This is an area that I haven't worked in at all before, so assume zero knowledge (but more-than-zero ability to understand ;) ).

Reply:
We're testing transactions fetched *through* the activity association, so therefore they have to belong to the 'correct activity', surely?

Reply:
That's what I don't get - in the hypothetical world where something went wrong, it could be that one of the transactions was assigned to another activity. Isn't that why we were previously testing the descriptions?

For example, the £5000 transaction could end up in the `project` instead of the `sibling_project`, and then we'd have a `"Q4 1999-2000 spend on Sibling Project"` showing up in `project.transactions.map(&:description).uniq`. I don't know how that could happen, but well, the spec was there, so I wanted to modify it to work with the new descriptions while keeping its spirit intact. 😬 😅